axelor-account/src/main/resources/views/InvoicePayment.xml

```diff
@@ -47,6 +47,7 @@
 			<field name="bankOrder" if-module="axelor-bank-payment" showIf="bankOrder" form-view="bank-order-form" grid-view="bank-order-grid"/>
 			<field name="statusSelect" showIf="(paymentMode.typeSelect == 2 || paymentMode.typeSelect  == 9) &amp;&amp; paymentMode.inOutSelect == 2"/>
 			<field name="statusSelect" hideIf="(paymentMode.typeSelect == 2 || paymentMode.typeSelect  == 9) &amp;&amp; paymentMode.inOutSelect == 2" selection-in="[0,1,2]"/>
+			<field name="typeSelect" showIf="id"/>
```

Comment:
No need of showIf on typeSelect because it is already given on otherDetailsPanel